server/src/internalClusterTest/java/org/elasticsearch/search/geo/GeoDistanceIT.java

```diff
@@ -126,51 +125,69 @@ public void testDistanceScript() throws Exception {
-            .get();
-        Double resultDistance1 = searchResponse1.getHits().getHits()[0].getFields().get("distance").getValue();
-        assertThat(resultDistance1, closeTo(GeoUtils.arcDistance(src_lat, src_lon, tgt_lat, tgt_lon), 0.01d));
-
+        assertResponse(
+            prepareSearch().addStoredField("_source")
+                .addScriptField("distance", new Script(ScriptType.INLINE, CustomScriptPlugin.NAME, "arcDistance", Collections.emptyMap())),
+            response -> {
+                Double resultDistance = response.getHits().getHits()[0].getFields().get("distance").getValue();
+                assertThat(resultDistance, closeTo(GeoUtils.arcDistance(src_lat, src_lon, tgt_lat, tgt_lon), 0.01d));
+            }
+        );
         // Test doc['location'].planeDistance(lat, lon)
-        SearchResponse searchResponse2 = prepareSearch().addStoredField("_source")
-            .addScriptField("distance", new Script(ScriptType.INLINE, CustomScriptPlugin.NAME, "planeDistance", Collections.emptyMap()))
-            .get();
-        Double resultDistance2 = searchResponse2.getHits().getHits()[0].getFields().get("distance").getValue();
-        assertThat(resultDistance2, closeTo(GeoUtils.planeDistance(src_lat, src_lon, tgt_lat, tgt_lon), 0.01d));
-
+        assertResponse(
+            prepareSearch().addStoredField("_source")
+                .addScriptField(
+                    "distance",
+                    new Script(ScriptType.INLINE, CustomScriptPlugin.NAME, "planeDistance", Collections.emptyMap())
+                ),
+            response -> {
+                Double resultDistance = response.getHits().getHits()[0].getFields().get("distance").getValue();
+                assertThat(resultDistance, closeTo(GeoUtils.planeDistance(src_lat, src_lon, tgt_lat, tgt_lon), 0.01d));
+            }
+        );
         // Test doc['location'].geohashDistance(lat, lon)
-        SearchResponse searchResponse4 = prepareSearch().addStoredField("_source")
-            .addScriptField("distance", new Script(ScriptType.INLINE, CustomScriptPlugin.NAME, "geohashDistance", Collections.emptyMap()))
-            .get();
-        Double resultDistance4 = searchResponse4.getHits().getHits()[0].getFields().get("distance").getValue();
-        assertThat(
-            resultDistance4,
-            closeTo(
-                GeoUtils.arcDistance(src_lat, src_lon, Geohash.decodeLatitude(tgt_geohash), Geohash.decodeLongitude(tgt_geohash)),
-                0.01d
-            )
+        assertResponse(
+            prepareSearch().addStoredField("_source")
+                .addScriptField(
+                    "distance",
+                    new Script(ScriptType.INLINE, CustomScriptPlugin.NAME, "geohashDistance", Collections.emptyMap())
+                ),
+            response -> {
+                Double resultDistance4 = response.getHits().getHits()[0].getFields().get("distance").getValue();
```

Comment:
nit: you've dropped the suffix from the other vars, maybe do this one too?

```suggestion
                Double resultDistance = response.getHits().getHits()[0].getFields().get("distance").getValue();
```

Reply:
done